ui/app/services/wizard.js

```diff
@@ -220,46 +267,49 @@ export default Service.extend(DEFAULTS, {
 
   saveFeatures(features) {
     this.set('featureList', features);
-    this.saveExtState(STORAGE_KEYS.FEATURE_LIST, this.get('featureList'));
+    this.saveExtState(STORAGE_KEYS.FEATURE_LIST, this.featureList);
     this.buildFeatureMachine();
   },
 
   buildFeatureMachine() {
-    if (this.get('featureList') === null) {
+    if (this.featureList === null) {
       return;
     }
     this.startFeature();
-    if (this.storageHasKey(STORAGE_KEYS.FEATURE_STATE)) {
-      this.saveState('featureState', this.getExtState(STORAGE_KEYS.FEATURE_STATE));
-    }
-    this.saveExtState(STORAGE_KEYS.FEATURE_STATE, this.get('featureState'));
-    let nextFeature =
-      this.get('featureList').length > 1
-        ? this.get('featureList')
-            .objectAt(1)
-            .capitalize()
-        : 'Finish';
+    let nextFeature = this.featureList.length > 1 ? this.featureList.objectAt(1).capitalize() : 'Finish';
     this.set('nextFeature', nextFeature);
     let next;
-    if (this.get('currentMachine') === 'secrets' && this.get('featureState') === 'display') {
-      next = FeatureMachine.transition(this.get('featureState'), 'REPEAT', this.get('componentState'));
+    if (this.currentMachine === 'secrets' && this.featureState === 'display') {
+      next = FeatureMachine.transition(this.featureState, 'REPEAT', this.componentState);
     } else {
-      next = FeatureMachine.transition(this.get('featureState'), 'CONTINUE', this.get('componentState'));
+      next = FeatureMachine.transition(this.featureState, 'CONTINUE', this.componentState);
     }
     this.saveState('nextStep', next.value);
-    let stateNodes = FeatureMachine.getStateNodes(this.get('featureState'));
+    let stateNodes = FeatureMachine.getStateNodes(this.featureState);
     this.executeActions(stateNodes.reduce((acc, node) => acc.concat(node.onEntry), []), null, 'feature');
   },
 
   startFeature() {
-    const FeatureMachineConfig = MACHINES[this.get('featureList').objectAt(0)];
+    const FeatureMachineConfig = MACHINES[this.featureList.objectAt(0)];
     FeatureMachine = Machine(FeatureMachineConfig);
-    this.set('currentMachine', this.get('featureList').objectAt(0));
-    this.saveState('featureState', FeatureMachine.initialState);
+    this.set('currentMachine', this.featureList.objectAt(0));
+    if (this.storageHasKey(STORAGE_KEYS.FEATURE_STATE)) {
+      this.saveState('featureState', this.getExtState(STORAGE_KEYS.FEATURE_STATE));
```

Comment:
What do you think about:
```suggestion
      this.saveState('featureState', this.getExtState(STORAGE_KEYS.FEATURE_STATE) || FeatureMachine.initialState);
```
Instead of this if/else?

Reply:
hmm not sure that GH suggestion will work since this would be a multi-line thing 😅